src/main/java/frc/robot/subsystems/elevator/ElevatorIOReal.java

```diff
@@ -123,35 +111,44 @@ public ElevatorIOReal() {
     hardwareLimitSwitchConfigs.ReverseLimitAutosetPositionEnable = true;
     hardwareLimitSwitchConfigs.ReverseLimitAutosetPositionValue = 0;
 
+    StatusCode rightStatus = StatusCode.StatusCodeNotInitialized;
+    for (int i = 0; i < 5; i++) {
+      rightStatus = m_backElevatorMotor.getConfigurator().apply(talonFXConfigs);
+      if (rightStatus.isOK()) break;
+    }
+    if (!rightStatus.isOK()) {
+      System.out.println("Could not configure device. Error: " + rightStatus.toString());
+    }
+
     motorOutput.Inverted = InvertedValue.Clockwise_Positive;
 
     StatusCode leftStatus = StatusCode.StatusCodeNotInitialized;
     for (int i = 0; i < 5; i++) {
-      leftStatus = m_leftElevatorMotor.getConfigurator().apply(talonFXConfigs);
+      leftStatus = m_frontElevatorMotor.getConfigurator().apply(talonFXConfigs);
       if (leftStatus.isOK()) break;
     }
     if (!leftStatus.isOK()) {
       System.out.println("Could not configure device. Error: " + leftStatus.toString());
```

Comment:
Consider using Alert class, like we do in AprilTagCam when the camera is not found.
Maybe kError instead of kWarning.

Might be good as a new PR?